bxsolana/utils/bundletip.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    Keypair,
+    PublicKey,
+    Transaction,
+    TransactionInstruction,
+    MessageCompiledInstruction,
+    VersionedTransaction,
+    SystemProgram
+} from "@solana/web3.js"
+
+const TRADER_API_TIP_WALLET = "AFT8VayE7qr8MoQsW3wHsDS83HhEvhGWdbNSHRKeUDfQ"
```

Comment:
I would not hardcode the but Document it and let the user get the most fresh one from our Docs.